### ISSUE TITLE
[Feature] Carry redstone signal over

### ISSUE THREAD

**Issue by samuelskovbakke** [SuperMartijn642/Entangled1.12#90]
I was trying to use entangled blocks to do wireless redstone control, but to my surprise this did not work.

I think this feature would be very nice to have in future versions of the mod. 

Expected result:
![image](https://github.com/user-attachments/assets/4573efe1-c24a-43b3-8ae8-8a02fd9fdaba)

Actual result:
![image](https://github.com/user-attachments/assets/8eb3931a-f705-4b19-b4e1-43cd585c3cac)

**Comment by SuperMartijn642** [comment]
This is not possible with the way Minecraft works, only a block itself can emit a redstone signal.
It does work the other way around though. If you bind the entangled block to something which emits a redstone signal or which gives a comparator output, the entangled block will match it

**Comment by samuelskovbakke** [comment]
By that logic should an entangled block that can be controlled with a redstone signal, e.g. on/off, not respond to a lever on the entangled block itself? Because that also does not seem to work... E.g I entangle a any machine that can be turned off with a redstone signal and then I put a lever on the entangled block, however, this does not seem to do anything.

Am I understanding your response incorrectly?

**Comment by SuperMartijn642** [comment]
No, the problem is that in order to give a block a redstone signal, it needs to have a block outputting that redstone signal on one of its six sides. The entangled block has no controll over what's around the bound block, hence that is not possible.

What is possible, is that if you bind the entangled block to a block which emits a redstone signal, like a lever or a redstone block, then the entangled block can emit the same signal to blocks around itself.

**Comment by Blockguy24** [comment]
I noticed that the Integrated Dynamics mod has an ability to remotely transmit a redstone signal without needing a block next to the target block outputting the signal. This is done with a "redstone writer" component, with an offset configured so that the signal is applied to a block that isn't immediately in front of it.

![Image](https://github.com/user-attachments/assets/55563fbd-831c-461e-9d77-6f50bb400891)

![Image](https://github.com/user-attachments/assets/45198683-7c68-405e-ac84-c558309cade2)

It appears that they implemented this somehow with mixins, here are some links to files that seem relevant:
- https://github.com/CyclopsMC/IntegratedDynamics/blob/e1fc7c1bb8cd4ffa457a214d41763b618e3650aa/src/main/java/org/cyclops/integrateddynamics/part/aspect/write/redstone/WriteRedstoneComponent.java
- https://github.com/CyclopsMC/IntegratedDynamics/blob/e1fc7c1bb8cd4ffa457a214d41763b618e3650aa/src/main/java/org/cyclops/integrateddynamics/mixin/MixinBlockStateBase.java

**Comment by DreamingLamb** [comment]
I came here to ask for this too..

Would it be possible to add a special case where the Entangled Block can create a ghost block next to the one that it's entangled with, and that ghost block emits a redstone signal?  To prevent conflicting block issues, maybe check which side the signal is coming from (North for example), then checking if the Northern block from the entangled block is air, if it is, create ghost redstone block, if it's not, do nothing?  This would rely on the player to provide redstone signals in the correct location though.

Or.. I'd be ok with having two entangled blocks, one for the block I want to affect and another for a ghost block?  For example, machine at 0,0,0 and ghost entangled block at 0,0,1, as the block at 0,0,1 has no data, this could be more easily replaced with a ghost redstone block.

My concern for doing this with the actual entangled block would be the amount of potential nbt data to replicate and not break could be insane.  Also seems that there's multiple nbt values that would affect whether something is powered..  powered, lit, triggered....

I know this might not be possible with the mod as it is, but ideas are ideas and might trigger other ideas that could be possible (like the Int Dyn's one above)

**Comment by thelapisfox** [comment]
I can confirm that Integrated Dynamics is still able to do this. Thank you, @Blockguy24, for pointing out the information about Integrated Dynamics.
